### ISSUE TITLE
Are the parameters of swinTransformer trained during fine-tuning?

### ISSUE THREAD

**Issue by DYF-AI** [clovaai/donut#208]
Are the parameters of swinTransformer trained during fine-tuning?

**Comment by gwkrsrch** [comment]
@DYF-AI Yes, during the fine-tuning phase, the vision encoder is trained as well.

**Comment by DYF-AI** [comment]
> @DYF-AI Yes, during the fine-tuning phase, the vision encoder is trained as well.

Thank you for your answer, and one more question, is there any code that provides training the Vision Encoder?
